### PR TITLE
Add ODM test

### PR DESCRIPTION
**Pull request by llxia**
- override existing run_ilog_with_gcmv.sh because the original
file expects certain Java path

Signed-off-by: lanxia <lan_xia@ca.ibm.com>

### CODE REVIEW

**Comment by llxia** [comment]
ODM tested on test-packet-ubuntu1604-x64-4: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/TestBuild_Sandbox2/223/

jdk_math: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/TestBuild_Sandbox2/224/

File: performance/odm/scripts/run_ilog_with_gcmv.sh [hunk 1]
```diff
@@ -0,0 +1,206 @@
+#!/bin/bash
+
```
Comment (smlambert) [inline performance/odm/scripts/run_ilog_with_gcmv.sh]: Need to add apache license here also.

File: performance/odm/scripts/benchmark.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+echo "***** Running Benchmark Script *****"
+
+export PRODUCT=pxa6480sr6
```
Comment (smlambert) [inline performance/odm/scripts/benchmark.sh]: eventually may clean up more of the odd references, but good for a start

**Comment by llxia** [comment]
Thanks Shelley. Copyright is added.

File: performance/odm/playlist.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
+	<test>
+		<testCaseName>odm</testCaseName>
+		<command>chmod a+x ./scripts/benchmark.sh; \
```
Comment (sophia-guo) [inline performance/odm/playlist.xml]: are those tests have platform requirement?

Comment (llxia) [reply performance/odm/playlist.xml]: Perf test needs run on specific machine. So far I only have linux to test. The scripts and cmd need to be updated as we test on other platforms.

Comment (sophia-guo) [reply performance/odm/playlist.xml]: Sounds good. I was thinking if <platformRequirements> is needed.

Comment (llxia) [reply performance/odm/playlist.xml]: sure, it can be add it.

**Pull request merged**